File: java/scripts/comandos/kick.java
// Command kick
// Exemplo: !kick <member>
// Linguagem usada: java
// Author: aquelemesmoojack

import java.time.temporal.ChronoUnit;

String[] args = event.getMessage().getContentRaw().split("\\s+");

if (args[0].equalsIgnoreCase(Main.prefix + "kick")) {
    if(args.length==1){
        event.reply("Você precisa especificar um membro para expulsar!").queue();
        return;
    }
    else if(event.getMentionedMembers().isEmpty()){
        event.getGuild().kick(args[1],0,"kick command").queue();
    }else{
        event.getGuild().kick(event.getMentionedMembers().get(0),0,"kick command").queue();
    }
}

File: lua/scripts/comandos/Kick.lua
--Kick command
--Exemplo: !kick <@member> <motivo>
--Linguagem usada: lua
--Author: aquelemesmoojack

function Kick:__init(data, parent)
    __init(self,data,parent)
    self.__init = self.client.__users:__insert(data.user)
end

function Kick:__hash()
    return self.__user.__id
end

function get.guild(self)
    return self.__parent
end

function get.user(self)
    return self.__user
end

return Kick